### ISSUE TITLE
Add multiple domains at once

### ISSUE THREAD

**Issue by McBochi** [Kong/kong-plugin-acme#16]
Hey guys,

it might sound stupid, but I am struggling to add more than one domain at once.
I tried almost every possible combination for the input data:
- [domain1.com, domain2.com]
- domain1.com, domain2.com
- domain1.com domain2.com
- {domain1.com, domain2.com}
..and all of it also encapsulated in quotes.

All I ever get is a error message like this:
```json
{
    "message": "schema violation (config.domains: {\n  \"invalid hostname: domain1.com domain2.com\"\n})",
    "name": "schema violation",
    "fields": {
        "config": {
            "domains": [
                "invalid hostname: domain1.com domain2.com"
            ]
        }
    },
    "code": 2
}
```

I also checked the code for all the schema definitions and also the Kong source code to find out more about the expected format, but can't even find something useful there...

What do I have to do to get a result similar to the situation described in https://github.com/Kong/kong-plugin-acme/issues/11 ?

**Comment by fffonion** [comment]
Hi, if you are using form-encoded then config.domains[] can be specified multiple times:
```
curl http://localhost:8001/plugins \
        -d name=acme \
        -d config.account_email=yourname@yourdomain.com \
        -d config.tos_accepted=true \
        -d config.domains[]=domain1.com \
        -d config.domains[]=domain2.com
```
or you can just post the config as a JSON.

I'm not sure what you mean by similar result as #11, could you elaborate more please?

**Comment by McBochi** [comment]
@fffonion Thanks for the eye opener! Feeling a bit stupid right know...
I also tried the request that way but I was using Postman with the body type set to "form-data". However, this always resulted in only the second domain being stored by kong.
I changed the request to "x-www-form-urlencoded" and it works!

Result when requesting the plugin status now looks like seen in #11 - this is what I referred to:
```
        "domains": [
            "domain1.com",
            "domain2.com"
        ],
```

Hopefully my stupid oversight here helps at least some others out there!
